MOxUnit/util/moxunit_util_escape_xml.m

```diff
@@ -0,0 +1,33 @@
+function str_esc=moxunit_util_escape_xml(str)
```

Comment:
Should the &gt; be a literal > instead?

On 13 December 2015 at 17:06, Nikolaas N. Oosterhof <
notifications@github.com> wrote:

> In MOxUnit/util/moxunit_util_escape_xml.m
> <https://github.com/MOxUnit/MOxUnit/pull/14#discussion_r47447311>:
>
> > @@ -0,0 +1,33 @@
> > +function str_esc=moxunit_util_escape_xml(str)
>
> I cloned your repo and ran your test, and get the following xml output:
>
> <testcase classname="my_test_of_abs" name="test_abs_error" time="0.005">
>   <failure message="Error using my_test_of_abs&gt;test_abs_error (line 19)
> elements are not equal
>
> First input:
> 7
>
> Second input:
> 8
> ">  test_abs_error:19 (/Users/nick/organized/tmp/matlab-ci/tests_moxunit/my_test_of_abs.m)
>   run:24 (/Users/nick/git/MOxUnit/MOxUnit/@MOxUnitFunctionHandleTestCase/run.m)
>   run:25 (/Users/nick/git/MOxUnit/MOxUnit/@MOxUnitTestSuite/run.m)
>   run:25 (/Users/nick/git/MOxUnit/MOxUnit/@MOxUnitTestSuite/run.m)
>   moxunit_runtests:78 (/Users/nick/git/MOxUnit/MOxUnit/moxunit_runtests.m)
>   runAllTests:19 (/Users/nick/organized/tmp/matlab-ci/runAllTests.m)</failure>
> </testcase>
>
> That looks fine, doesn't it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MOxUnit/MOxUnit/pull/14/files#r47447311>.
>